src/main/java/io/github/hildi/can/service/StandardJavaSerializationService.java

```diff
@@ -0,0 +1,34 @@
+package io.github.hildi.can.service;
+
+import io.github.hildi.can.exceptions.CannotReadFileException;
+import io.github.hildi.can.model.User;
+
+import java.io.*;
+
+/**
+ * Created by Serhii Hildi on 15.03.19.
+ */
+public class StandardJavaSerializationService implements SerializationService {
+
+    @Override
+    public void serialize(User user, File file) {
+
+        if (!(file.exists() || file.canWrite())) { return; }
+        try (ObjectOutputStream out = new ObjectOutputStream(new FileOutputStream(file))){
+            out.writeObject(user);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
1) you've simply swallowed exception here, it's not good at all, we'll discuss why later.
2) take a look at [this link](https://stackoverflow.com/questions/7469316/why-is-exception-printstacktrace-considered-bad-practice), why `printStackTrace()` is a bad practice.

Reply:
Done